book/src/specs/kimchi.md

```diff
@@ -1969,6 +1969,14 @@ pub struct ProofEvaluations<Evals> {
     pub generic_selector: Evals,
     /// evaluation of the poseidon selector polynomial
     pub poseidon_selector: Evals,
+    /// evaluation of the EC addition selector polynomial
+    pub complete_add_selector: Evals,
```

Comment:
So this means only the purely optional gates such as Rot64 will be part of the linearization process? Got it